src/API/Google/Merchant.php

```diff
@@ -314,4 +314,15 @@ public function has_access( string $email ): bool {
 
 		return false;
 	}
+
+	/**
+	 * Update the Merchant Center ID to use for requests.
+	 *
+	 * @param int $id  Merchant ID number.
+	 *
+	 * @return bool
+	 */
+	public function update_merchant_id( int $id ): bool {
+		return $this->options->update( OptionsInterface::MERCHANT_ID, $id );
```

Comment:
Nice catch. As long as the unit tests cover that line of code, it will produce an error. Seems like I forgot to add the test for that function though. I already had the test for `update_ads_id`, so I just copied that into MerchantTest